### ISSUE TITLE
Can we globally turn on the setting "Send me email whenever someone comments on my post" for all users?

### ISSUE THREAD

**Issue by cimenta** [Erudika/scoold#430]
hi

Can we globally turn on the setting "Send me email whenever someone comments on my post" for all users?

We want to turn on the approval processs for posts to make sure that the quality of questions is high. If new questions needs to be edited by the author we would comment on the question instead of approving it.

It would be great to know that the author was notified about our comments.

Thank you
Radek

**Comment by albogdano** [comment]
Yes, but that doesn't mean that users can't turn them back off if they want.
```ini
scoold.comment_emails_enabled = true
```
